plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryAvroConnectorTest.java

```diff
@@ -14,11 +14,32 @@
 package io.trino.plugin.bigquery;
 
 import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
 import io.trino.testing.QueryRunner;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.Optional;
+import java.util.Set;
+
+import static io.trino.testing.DataProviders.toDataProvider;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 public class TestBigQueryAvroConnectorTest
         extends BaseBigQueryConnectorTest
 {
+    // these column names started to fail because of BigQuery internal change ~ between 18:00 16 of March 2023 GMT and 00:00 17 of March 2023 GMT
+    // it's still possible to CREATE TABLE with such column names, and even INSERT some data, but it is not possible to SELECT from it.
```

Comment:
> but it is not possible to SELECT from it.

This sentence is misleading. We can select the table using query based API. https://github.com/trinodb/trino/pull/16613#discussion_r1141524457